tests/benchmark/Dockerfile.in

```diff
@@ -1,4 +1,6 @@
-FROM ${IMAGE_NAME}
+FROM ${FULL_BASE_IMAGE}
 
-RUN hg clone https://hg.python.org/benchmarks /app/benchmarks
+RUN hg clone -r 9923b81a1d34 https://hg.python.org/benchmarks /app/benchmarks
```

Comment:
The package went from deprecated to outright deleted, so I clone from just before the deletion.  #62 tracks migrating to a different benchmarking framework.